docs/infrastructure/installation/system-requirements.md

```diff
@@ -13,7 +13,7 @@ labels:
 For reliable performance in production environments, it is recommended to run an XRP Ledger (`rippled`) server on bare metal with the following characteristics or better:
 
 - Operating System: Ubuntu (LTS), Red Hat Enterprise Linux (latest release), or a compatible Linux distribution.
-- CPU: Intel Xeon 3+ GHz processor with 8+ cores and hyperthreading enabled.
+- CPU: 64-bit x86_64, 8+ cores.
```

Comment:
```suggestion
- CPU: 3+ GHz 64-bit x86_64 processor with 8+ cores.
```

Servers are rarely CPU-bound, but this CPU requirement has not changed since at least 2020. While other brands (like AMD) are fine, the processors still exceed 3 GHz.